### PR TITLE
Fix spellings in Master

### PR DESCRIPTION
**Pull request by samgnaniah**
Fixes spelling issues in master branch that are raised by the tool and removing extra spelling files.

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/adapter-configurations.md [hunk 1]
```diff
@@ -396,7 +396,7 @@ See the example .toml file given below.
                     <div class="mb-config-example">
 <pre><code class="toml">[adapter.consul]
   enable = false
-  url = "https://169.254.1.1:8501" # scheme + host ip + port
+  URL = "https://169.254.1.1:8501" # scheme + host ip + port
```
Comment (Mariangela) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/adapter-configurations.md:399]: It is incorrect to have URL in upper case here.

File: en/docs/install-and-setup/setup/setting-up-databases/working-with-the-resgistry/using-remote-registry/admin-config-and-governance-partitions-in-separate-nodes.md [hunk 2]
```diff
@@ -336,7 +336,7 @@ Change the values of the relevant elements according to your environment. ******
 
 ***Configuring registry.xml file***
 
-2. Navigate to $CARBON\_ HOME/repository/conf/registry.xm l file and specify the following configurations.
+2. Navigate to $CARBON\_ HOME/repository/conf/registry.xml file and specify the following configurations.
```
Comment (Mariangela) [inline en/docs/install-and-setup/setup/setting-up-databases/working-with-the-resgistry/using-remote-registry/admin-config-and-governance-partitions-in-separate-nodes.md:339]: Shall we replace $CARBON\_ HOME with <PRODUCT_HOME>  ?

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@samgnaniah please see my comments.

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/adapter-configurations.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/adapter-configurations.md]: Good catch. Change reverted.

**Pull request merged**